src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/index.js

```diff
@@ -47,23 +44,32 @@ const IncidentSplitForm = ({ parentIncident, subcategories, directingDepartments
             <dd data-testid="incidentSplitFormSubcategoryDisplayName">{parentIncident.subcategoryDisplayName}</dd>
           </StyledDefinitionList>
 
-          <IncidentSplitRadioInput
-            display="Regie"
-            register={register}
-            initialValue={parentIncident.directingDepartment}
-            name="department"
-            id="department"
-            data-testid="incidentSplitFormRadioInputDepartment"
-            options={directingDepartments}
+          <StyledBottomWrapper>
```

Comment:
I would add this bottom margin as a style on the `IncidentSplitRadioInput`.

Reply:
I've refactored the implementation for styling this element, please take another look :)